sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -348,6 +358,8 @@ def aggregate(self, func, axis=0, *args, **kwargs):
       'order-sensitive')
   diff = frame_base.wont_implement_method('order-sensitive')
 
+  drop = frame_base._elementwise_method('drop')
```

Comment:
drop has a lot of arguments, not all of them work with elementwise.

Reply:
Ok we can just drop that commit for now. I'm not actually using drop in the demo since `__getitem__` is cleaner.

Reply:
Done